src/App.jsx

```diff
@@ -8,52 +8,59 @@ import EOSIconsList from "./converted-icons.json";
 import OptionsList from "./optionList.json";
 import IconBox from "./IconBox";
 
-const copyToClipboard = (svg) => {
-  copyText(ReactDOMServer.renderToStaticMarkup(svg));
-};
-
-const searchIconsByName = (name, theme, option) => {
-  const iconDivs = [];
-  EOSIconsList[option].forEach((icon) => {
-    const isFilledAvailable = "hasOutlined" in icon && icon.hasOutlined;
-    const isFilledSelected = theme === "Filled";
-    if (
-      icon.name.indexOf(name) !== -1 &&
-      (!isFilledSelected || (isFilledSelected && isFilledAvailable))
-    ) {
-      const nameIcon = `Eos${icon.name}${
-        isFilledSelected ? "Filled" : "Outlined"
-      }`;
-      const EOSReactIcon = EOSIcons[nameIcon];
-      if (EOSReactIcon !== undefined) {
-        iconDivs.push(
-          <div
-            className="image-container"
-            key={nameIcon}
-            onClick={() => {
-              copyToClipboard(EOSReactIcon({ size: "xxxl" }));
-            }}
-          >
-            {EOSReactIcon({ size: "xl" })}
-          </div>
-        );
-      }
-    }
-  });
-  if (iconDivs.length === 0) {
-    return null;
-  }
-  return <IconBox option={option} iconDivs={iconDivs} />;
-};
-
 const App = () => {
   const searchCategory = useRef();
   const searchTheme = useRef();
   const inputField = useRef();
   const [helperText, setHelperText] = useState(
     <p>Let&apos;s start by searching abstract.</p>
   );
+  const [alert, updateAlert] = useState(null);
   const [iconsContainer, updateIcons] = useState(null);
+  const copyToClipboard = (svg) => {
+    copyText(ReactDOMServer.renderToStaticMarkup(svg));
+    updateAlert(
+      <div className="alert">
+        <span>Svg has been copied!!</span>
+      </div>
+    );
+    setTimeout(() => {
+      updateAlert(null);
+    }, 1000);
+  };
+  const searchIconsByName = (name, theme, option) => {
+    const iconDivs = [];
+    EOSIconsList[option].forEach((icon) => {
+      const isFilledAvailable = "hasOutlined" in icon && icon.hasOutlined;
+      const isFilledSelected = theme === "Filled";
+      if (
+        icon.name.indexOf(name) !== -1 &&
+        (!isFilledSelected || (isFilledSelected && isFilledAvailable))
+      ) {
+        const nameIcon = `Eos${icon.name}${
+          isFilledSelected ? "Filled" : "Outlined"
+        }`;
+        const EOSReactIcon = EOSIcons[nameIcon];
+        if (EOSReactIcon !== undefined) {
+          iconDivs.push(
+            <div
+              className="image-container"
+              key={nameIcon}
+              onClick={() => {
+                copyToClipboard(EOSReactIcon({ size: "xxxl" }));
+              }}
+            >
+              {EOSReactIcon({ size: "xl" })}
+            </div>
+          );
+        }
+      }
+    });
+    if (iconDivs.length === 0) {
+      return null;
+    }
+    return <IconBox option={option} iconDivs={iconDivs} key={option} />;
```

Comment:
shouldn't we pass something like icons array to this component and render them there instead of passing an array of divs? (referring to iconsDivs)
if not, make it a separate component at least. The purpose for this is whenever we make some style changes, we should know which component to update rather than looking for html being injected through functions.